routes/api/v1/cgTranscripts.js

```diff
@@ -0,0 +1,110 @@
+var express = require("express");
+var router = express.Router();
+var mongoose = require("mongoose");
+var fq = require("fuzzquire");
+var users = fq("users");
+var config = fq("config");
+var path = require("path");
+var fs = require("fs");
+let appRoot = require("app-root-path");
+var errorFields = [];
+var cgTranscriptsModel = require("../../../schemas/cgTranscripts");
+
+let authenticate = function(req, res, next) {
+  if (true) {
+    return next();
+  } else {
+    res.status(401).json({
+      error: true,
+      message: "Uable to authenticate"
+    });
+    return;
+  }
+};
+
+let checkErrors = function(req, res, next) {
+  if (errorFields.length > 0) {
+    res.status(400).json({
+      error: true,
+      message: `Invalid fields ${errorFields}`
```

Comment:
Why not pass entire `errorFields` as array itself and let the client component take care of it?